website/content/en/v1.0/reference/settings.md

```diff
@@ -49,6 +49,10 @@ Karpenter uses [feature gates](https://kubernetes.io/docs/reference/command-line
 | Drift                   | true    | Beta  | v0.33.x |         |
 | SpotToSpotConsolidation | false   | Alpha  | v0.34.x |         |
 
+{{% alert title="Note" color="primary" %}}
```

Comment:
Can we update the table above too so that it marks that it's no longer beta and it's stable now?

Reply:
I somehow feel, updating the table would be misleading because it's not a feature gate anymore and the Note should be self-explanatory,. What do you think?

Reply:
I still think updating the table makes sense. At a minimum, we should update the section that says "until" on beta since right now it looks like the feature is still in beta if I was glancing at the table.

Reply:
@jonathan-innis I updated the table according to your suggestions including the budget drift example in doc, preview and v1.0 folder.